vault/values.yaml

```diff
@@ -199,3 +199,6 @@ statsd:
       labels:
         method: "$1"
         path: "$2"
+
+podSecurityPolicy:
```

Comment:
@anasinnyk our convention is the following: we put PSP under RBAC key, and we set it to false by default. Could you add rbac and psp values to the chart you can see below? I know, rbac value is missing now but we will change it in case of this chart as well. So your solution will be future compatible.
```
rbac:
  enabled: true
  psp:
    enabled: false
```

Reply:
@pbalogh-sa please check my last commit. I try to re-implement it by your standard.